portfolio/src/main/webapp/comments.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>OliviaA: Photos </title>
+        <link rel="stylesheet" href="style.css">
+        <script src="script.js"></script>
+    </head>
+
+    <body onload="getCommentFromServer()">
+
+    <header>
+        <h1>Comments</h1>
+        <p> Comments, Questions, Concerns? Ask down below!</p>
+    </header>
+
+    <div class="menu">
+      <nav>
+        <a href="index.html">Home</a>
+        <a href="about.html"> About </a>
+        <a href="map.html"> Map </a>
+        <a href="chart.html"> Chart </a>
+        <a href="gallery.html"> Photos </a>
+      </nav>
+    </div>
+
+      <form id="form" action="/commentServ" method="POST" >
+        <p>Questions, Comments, Concerns? Say it down below!</p>
+        <label for="name"> Name: </label> <br>
+        <input type="text" id="name" name="name"> <br>
+
+        <label for="comment">Comment: </label> <br>
+        <textarea id="comment" name="comment" placeholder="Write your comment here" rows="5" cols="50"></textarea>
+        <br> <br>
+        <input type="submit" value="Submit">
+      </form>
+
+      <div id="comments-container"></div>
+
+    <!--
```

Comment:
Remove commented out HTML? It may be for a future PR, but it's better to `git stash` the change rather then leaving it commented out. This way, if you never implement the file upload, the commented out code won't be confusing.